Public/Src/Cache/ContentStore/Library/Service/Grpc/GrpcContentServer.cs

```diff
@@ -319,18 +311,18 @@ private async Task<(long Chunks, long Bytes)> StreamContentAsync(Stream reader,
                 chunks++;
 
                 // Read the next chunk while waiting for the response
-                await Task.WhenAll(ReadNextChunk(), responseStream.WriteAsync(response));
+                await Task.WhenAll(readNextChunk(), responseStream.WriteAsync(response));
             }
 
             return (chunks, bytes);
 
-            async Task<int> ReadNextChunk() { chunkSize = await reader.ReadAsync(buffer, 0, buffer.Length, ct); return chunkSize; }
+            async Task<int> readNextChunk() { chunkSize = await input.ReadAsync(buffer, 0, buffer.Length, ct); return chunkSize; }

```

Comment:
Is it convention to name local functions starting with lower case?

Reply:
Yep.